pkgs/tools/networking/grpc_cli/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, automake, cmake, autoconf, curl, numactl }:
+
+stdenv.mkDerivation rec {
+  pname = "grpc_cli";
+  version = "1.46.6";
+
+  src = fetchFromGitHub {
+    owner = "grpc";
+    repo = "grpc";
+    rev = "v${version}";
+    hash = "sha256-m55FKwqlAQ2Ge7y4RMFO4W1onu0NYARWrOygChJMxng=";
+    fetchSubmodules = true;
+  };
+  buildInputs = [ automake cmake autoconf curl numactl ];
```

Comment:
Some of these are apparently nativeBuildInputs (which is for compilers/build-time tools), buildInputs is only for libraries (run-time).

Reply:
Moved automake, cmake and autoconfg to nativeBuildInputs.